### DISCUSSION TITLE
Need trivy reports in csv format (excel)

### DISCUSSION THREAD

**Discussion by rajeev-cs** [aquasecurity/trivy#7692] [Ideas]
### Description

It would be great if you develop a feature to get the trivy scan reports in csv format as well.

### Target

None

### Scanner

None

**Comment by itaysk** [comment]
there are no plans to add more output formats into Trivy, this can be created as an external output plugin

**Comment by nikpivkin** [comment]
Hi @rajeev-cs !

You can write your own plugin to convert scan results to csv. See documentation https://aquasecurity.github.io/trivy/v0.56/docs/configuration/reporting/#plugin
